config/environments/development.rb

```diff
@@ -66,7 +66,7 @@
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  config.action_mailer.default_url_options = { host: (ENV["DOMAIN"] || "local.abscond.org"), port: 3000 }
+  config.action_mailer.default_url_options = { host: (ENV["DOMAIN"] || "local.lvh.me"), port: 3000 }
```

Comment:
We don't need the `local` bit in this, that was just because I set it as the local subdomain on my domain name. `lvh.me` will work on it's own.